### DISCUSSION TITLE
Tiffslide-Xarray

### DISCUSSION THREAD

**Discussion by swamidass** [pydata/xarray#8354] [Show and tell]
I've made a new plugin for Xarray that uses tiffslide to read tiff and svs files.

https://github.com/swamidasslab/tiffslide-xarray

```
pip install tiffslide-xarray
```

Our group is using this library to process whole-slide images in Pathology.  Let us know if this is helpful to you and how we can better design the API before fully stabilizing it.

**Comment by jhamman** [comment]
🙌 Thanks for sharing @swamidass! 

If you like, we'd be happy to show off the tiffslide-xarray integration on in the Ecosystem page of the [Xarray docs](https://docs.xarray.dev/en/stable/ecosystem.html#extend-xarray-capabilities).

I'd be interested to learn a bit more about how you are using Xarray to process whole-slide images. Perhaps you can share more about your application? If you are up for it, we're always keen to have folks tell their use-case story on the xarray blog (https://github.com/xarray-contrib/xarray.dev/issues/344). 

Cheers (and thanks again for sharing)!

**Comment by swamidass** [comment]
Thanks for the invitation. I'm pretty swamped now, but maybe in a month our so the library will be stable enough for me to write a blog post about it. I'll let you know. At that point, adding it to the docs would be great.

How does that sound?

**Reply by dcherian** [reply]
Sounds great!
